tests/bugs/texture2d-ms.hlsl.expected

```diff
@@ -0,0 +1,55 @@
+result code = 0
```

Comment:
I don't think we need to check in this `.expected` file if there is already a `.hlsl.glsl` file like the one below. It shouldn't harm things to have it for now, since the testing process will clobber it, I think.